frontend/assets/style/_variables.scss

```diff
@@ -26,13 +26,6 @@ $size_extra-large: $size_1;
 $radius: 3px;
 $radius-large: 5px;
 
-// Breakpoints
-// // Same as JS breakpoints at breakpoints.js
-// TODO: Redefine the breakpoints with @mmbotelho in a next task.
-$gap: 1rem;
-$tablet: 769px;
-$desktop: 1200px;
-
```

Comment:
breakpoint variables have now been moved to `_mixins.scss` which is `@forward`ed by this file now. This ensures any `.vue` file that imports/uses `variables.scss` in its scoped style block can safely access these variables, just like before.